app/blueprints/graph.py

```diff
@@ -50,13 +50,15 @@
             rdfs:label ?target_label .
     ?activity rdfs:comment ?source_label .
     FILTER NOT EXISTS {{?activity rdf:type wfprov:WorkflowRun}}
+    FILTER EXISTS {{?activity rdf:type wfprov:ProcessRun}}
```

Comment:
Wasn't sure if you added those on purpose?

Reply:
yes, I have added it on @lorenzo-cavazzi PC